js/ui/status/automaticUpdates.js

```diff
@@ -171,6 +199,11 @@ var Indicator = new Lang.Class({
 
         // The status label
         this._item.label.text = _("Automatic Updates");
+
+        this._state = this._getState();
+
+        this._updateTimeoutId = 0;
+        return GLib.SOURCE_REMOVE;
```

Comment:
Move these too lines into the arrow function, this function shouldn't become a GSourceFunc just because it's now used with `timeout_add_seconds`